packages/wonder-blocks-form/src/components/__docs__/checkbox.stories.js

```diff
@@ -0,0 +1,156 @@
+// @flow
```

Comment:
One general suggestion that came from @megaroeny in this ticket: https://khanacademy.atlassian.net/browse/WB-1278

> Remove redundant button(s) words
> Basic Buttons → Basic or combine with Default
> Buttons with Colors → with Colors
> Buttons with Size → Sizes

What do you think about this?

(Sorry that I didn't mention this before, I just started using this pattern with `Clickable`).

Reply:
Interesting! I'll go ahead and update my titles to match this.

I'm not personally bothered by the redundancy. I actually think it could be helpful if you have a lot of the Storybook directories open so it's clearer which directory you're under.

Reply:
That's a good point. I took a couple of screenshots with multiple subdirs open and also using the filter option, and I think the short versions might have sense. Another point is that the stories in the `Docs` page look better with the shorten version. Well it's my personal opinion :), so I'm not sure what y'all think.

<img width="1335" alt="Screen Shot 2022-05-05 at 5 19 59 PM" src="https://user-images.githubusercontent.com/843075/167028161-9486d8ae-3784-4b09-9a9f-9f5d62504abd.png">

<img width="263" alt="Screen Shot 2022-05-05 at 5 21 43 PM" src="https://user-images.githubusercontent.com/843075/167028180-4d3b4e6d-950a-476e-a368-c8c091d9949b.png">

I'm fine with either option. We'll see how folks react to the docs.